src/main/archetype/ui.frontend.angular/README.md

```diff
@@ -40,3 +40,134 @@ Furthermore, an AEM ClientLib is generated from the app using the [`aem-clientli
 ${hash}${hash} Browser Support
 
 By default, this project uses [Browserslist](https://github.com/browserslist/browserslist)'s `defaults` option to identify target browsers. Additionally, it includes polyfills for modern language features to support older browsers (e.g. Internet Explorer 11). If supporting such browsers isn't a requirement, the polyfill dependencies and imports can be removed.
+
+
+
+#if ( $enableSSR == "y" )
+
+${hash} Frontend Serverless Build (Adobe IO Runtime)
+
+${hash}${hash} Features
+
+* AdobeIO Runtime (Serverless) rendering of the SPA editor 
```

Comment:
is this true? is it not only express here?

Reply:
Ah yep, good catch